storage/sql/migrate.go

```diff
@@ -229,4 +229,25 @@ var migrations = []migration{
 		},
 		flavor: &flavorMySQL,
 	},
+	{
+		stmts: []string{`
+			create table device_request (
+				user_code text not null primary key,
+				device_code text not null,
+				client_id text not null,
+				client_secret text ,
+				scopes bytea not null, -- JSON array of strings
```

Comment:
Is this kind of comment allowed in SQL statements like this? Never seen it before 😅

Reply:
@JoelSpeed  i actually copied this comment format from earlier in migrate.go.  it was used in creating the older tables :) (see client, auth_request, auth_code, refresh_token, and keys )